scenarios/csharp/dotnet/captioning/Program.cs

```diff
@@ -0,0 +1,417 @@
+/*
+Dependencies
+*/
+
+using System; // File
+using System.Linq; // Enumerable.Contains
+using System.Text; // StringBuilder
+using System.Threading.Tasks; // Task
+
+/* Notes:
+- Install the Speech NuGet package:
+dotnet add package Microsoft.CognitiveServices.Speech
+- The Speech SDK on Windows requires the Microsoft Visual C++ Redistributable for Visual Studio 2019 on the system. See:
+https://docs.microsoft.com/azure/cognitive-services/speech-service/speech-sdk?tabs=windows%2Cubuntu%2Cios-xcode%2Cmac-xcode%2Candroid-studio
+*/
+using Microsoft.CognitiveServices.Speech;
+using Microsoft.CognitiveServices.Speech.Audio;
+
+namespace Caption
+{
+/*
+Types
+*/
+
+    public struct UserConfig
+    {
+        readonly public bool profanityFilterRemoveEnabled = false;
+        readonly public bool profanityFilterMaskEnabled = false;
+        readonly public string[]? languageIDLanguages;
+        readonly public string? inputFile;
+        readonly public string? outputFile;
+        readonly public string? phraseList;
+        readonly public bool suppressOutputEnabled = false;
+        readonly public bool partialResultsEnabled = false;
+        readonly public string? stablePartialResultThreshold;
+        readonly public bool srtEnabled = false;
+        readonly public bool trueTextEnabled = false;
+        readonly public string subscriptionKey;
+        readonly public string region;
+        
+        public UserConfig (
+            bool profanityFilterRemoveEnabled,
+            bool profanityFilterMaskEnabled,
+            string[]? languageIDLanguages,
+            string? inputFile,
+            string? outputFile,
+            string? phraseList,
+            bool suppressOutputEnabled,
+            bool partialResultsEnabled,
+            string? stablePartialResultThreshold,
+            bool srtEnabled,
```

Comment:
A few of these, but especially this one, are not obvious from the name and for sure bear either a longer name (not sure "subripEnabled" is much better, but maybe "captionFileFormatEnabled" or something in that vein) and/or a comment